Update shell-integration.md (#4397)
wez/wezterm: docs/shell-integration.md

You can find some [examples for various shells in the wezterm
repo](https://github.com/wez/wezterm/tree/main/assets/shell-integration).

To use this file to setup shell integration in wezterm with Bash or Zsh, you can
copy the file to your computer and source it via `. /path/to/file.sh` in your `.bashrc`

> or `.zshrc`, or you can install it at `/etc/profile.d` if using Linux.

Reviewer: ```suggestion
or `.zshrc`, or you can install it at `/etc/profile.d` on most unix systems.
```
